tests/gnrc_sock_dns/main.c

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (C) 2017 Kaspar Schleiser <kaspar@schleiser.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     tests
+ * @{
+ *
+ * @file
+ * @brief       sock DNS client test application
+ *
+ * @author      Kaspar Schleiser <kaspar@schleiser.de>
+ *
+ * @}
+ */
+
+#include <stddef.h>
+#include <stdio.h>
+
+#include "net/sock/dns.h"
+#include "net/sock/util.h"
+#include "xtimer.h"
+
+#ifndef TEST_NAME
+#define TEST_NAME   "riot-os.org"
```

Comment:
Please choose a name not available on the Internet ;-). `www.example.org` [was created for that very purpose](https://tools.ietf.org/html/rfc6761#section-5)